MSAL/src/MSALWebviewType.m

```diff
@@ -40,3 +41,22 @@
     
     @throw @"Unrecognized MSALWebviewType";
 }
+
+extern MSIDWebviewType MSIDWebviewTypeFromMSALType(MSALWebviewType type)
+{
+    switch (type) {
+#if TARGET_OS_IPHONE
+        case MSALWebviewTypeDefault:
+            return MSIDWebviewTypeDefault;
+        case MSALWebviewTypeAuthenticationSession:
+            return MSIDWebviewTypeAuthenticationSession;
+        case MSALWebviewTypeSafariViewController:
+            return MSIDWebviewTypeSafariViewController;
+#endif
+        case MSALWebviewTypeWKWebView:
+            return MSIDWebviewTypeWKWebView;
+
+        default:
+            break;
```

Comment:
What happen if unexpected webview type is seen here? do we fail or set it to a default one?

Reply:
Fixed